drivers/amlogic/input/remote/remote_main.c

```diff
@@ -870,22 +870,25 @@ static int remote_resume(struct platform_device *pdev)
 	if (is_meson_m8m2_cpu()) {
 #define  AO_RTI_STATUS_REG2 ((0x00 << 10) | (0x02 << 2))
 		if (aml_read_aobus(AO_RTI_STATUS_REG2) == 0x1234abcd) {
+#ifndef CONFIG_GXBB_FORCE_POWER_ON_STATE_AFTER_RESUME
 			input_event(gp_remote->input, EV_KEY, KEY_POWER, 1);
 			input_sync(gp_remote->input);
 			input_event(gp_remote->input, EV_KEY, KEY_POWER, 0);
 			input_sync(gp_remote->input);
-
+#endif
 			/*aml_write_reg32(P_AO_RTC_ADDR0,
 			(aml_read_reg32(P_AO_RTC_ADDR0) | (0x0000f000)));*/
 			aml_write_aobus(AO_RTI_STATUS_REG2, 0);
 		}
 	} else {
 		if (get_resume_method() == REMOTE_WAKEUP) {
 			input_dbg("remote_wakeup\n");
+#ifndef CONFIG_GXBB_FORCE_POWER_ON_STATE_AFTER_RESUME
 			input_event(gp_remote->input, EV_KEY, KEY_POWER, 1);
 			input_sync(gp_remote->input);
 			input_event(gp_remote->input, EV_KEY, KEY_POWER, 0);
 			input_sync(gp_remote->input);
+#endif
 		}
```

Comment:
@codesnake Maybe because of different implementation. I was solving this issue on Buildroot using NetworkManager, not connmann. Point is if you issue ifconfig wlan0 down before suspend it will work just fine. I'm pretty sure it's same on LE/OE but connmann makes that difference (crashes even when not connected to AP).